src/components/Label/Label.tsx

```diff
@@ -0,0 +1,41 @@
+import { useStyles } from '@helpers/hooks/useStyles'
+import { FC, ReactNode } from 'react'
+
+import styles from './Label.module.scss'
+
+interface LabelProps {
+  children: ReactNode
+  className?: string | string[]
+  id?: string
+  isToggle?: boolean
+  toggle?: boolean
+  padding?: 'default' | 'none'
+}
+
+export const Label: FC<LabelProps> = ({
+  children,
+  className = '',
+  id,
```

Comment:
jest opcjonalny bez domyślnej wartości?

Reply:
<img width="583" alt="Zrzut ekranu 2022-12-13 o 12 21 45" src="https://user-images.githubusercontent.com/72187174/207304874-80ce1426-0c88-4326-aeab-3a18a21b20fc.png">

Jego domyślna wartość to null.